### DISCUSSION TITLE
How to reference custom-values.yaml file outside helm-chart (.tgz file) in Jfrog Artifactory

### DISCUSSION THREAD

**Discussion by premsolomon1690** [fluxcd/flux2#4561] [Q&A]
We are using Gitops model with Flux for deploying infrastructure components for more than 200 plus EKS clusters (single tenant). The helm-charts of the infrastructure components are sourced from helm repository in Jfrog Artifactory. These helm-charts are common across all our eks clusters. Few clusters require some additional customisations for these infrastructure components. We need to override the custom-values.yaml with the default values.yaml. Is this achievable with flux if the custom-values.yaml is placed outside of the .tgz (helm-chart) file and referenced inside the helmrelease manifest ?

Tools:

Flux version : 2.1.2
Registry: Jfrog Artifactory(helm repository)

**Comment by stefanprodan** [comment]
Create a ConfigMap or Secret in the clusters and in the HelmReleases use `valuesFrom`. The ConfigMap or Secret can be stored in Jfrog Artifactory as an OCI artifact created with `flux push artifact` and applied on the cluster with a `OCIRepository & Kustomization`.

**Reply by premsolomon1690** [reply]
Can you share some scenarios of storing Oci artifacts in Jfrog artifactory along with ocirepository and Kustimization templates ?

**Reply by premsolomon1690** [reply]
Do we have any roadmap to include this feature in the upcoming flux versions ?

**Reply by suryapandian** [reply]
where you able to push flux OCI artifact to jfrog? if yes, can you please share some insight?

```
kustomizer push artifact oci://xxx.jfrog.io/oci-development/gitops-watcher-app-test:1.0.0 -f /Users/suryapandian/repo/tl/gitops-watcher-app-test/deploy/releases/development.yaml
```


I keep getting the error

```
✗ pushing image failed: pushing image failed: GET https://xxx.jfrog.io/v2/: : Unsupported v2 repository request for ''
```
